accord-core/src/main/java/accord/primitives/Routables.java

```diff
@@ -46,6 +48,10 @@ enum Slice
     int size();
 
     boolean isEmpty();
+    default Stream<K> stream()
```

Comment:
this is dead code in the patch, but I was using it in debugger often; I can remove, just figured its useful...

Reply:
seems harmless enough